pallet/src/lib.rs

```diff
@@ -13,12 +13,20 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+mod migrations;
+
 use frame_support::Parameter;
 use sp_std::prelude::*;
 
-use frame_support::{sp_runtime::BoundToRuntimeAppPublic, traits::OneSessionHandler};
+use frame_support::{
+    sp_runtime::BoundToRuntimeAppPublic,
+    traits::{OneSessionHandler, StorageVersion},
+};
 pub use pallet::*;
 
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Unfortunately "current" is a little bit misleading in this context (at least for me) - as far as I understand, this corresponds to the storage version represented by the code, not the one on-chain, and will not be changed by a possible migration. Am I right?

Reply:
this is Parity parlance: https://crates.parity.io/frame_support/pallet_prelude/trait.GetStorageVersion.html
I figured it's best to stick with it (for consistency and stuff)